myfirstcatapi/domains/cat_domain.py

```diff
@@ -32,3 +35,12 @@ async def find_many(
 
 async def delete_one(cat_id: dto.CatID) -> dto.ResultCount:
     return await cat_model.delete_one(cat_id=cat_id)
+
+
+async def update_cat_metadata(cat_id: dto.CatID, url: dto.CatURL) -> dto.ResultCount:
```

Comment:
```suggestion
async def update_cat_metadata(cat_id: dto.CatID, url: dto.PartialUpdate) -> dto.ResultCount:
```
Passing `url` to this function means that it will be very hard to extend this in case other data also needs to be updated. I.e. what if we need to update the name, birthdate, gender, owner name etc?
Would you introduce all these fields as arguments? 
WDYT?

Reply:
done